test/NosCore.Tests/HandlerTests/InventoryPacketControllerTests.cs

```diff
@@ -179,8 +180,8 @@ public void Test_Put()
         [TestMethod]
         public void Test_PutBadPlace()
         {
-            _session.Character.PositionX = 3;
-            _session.Character.PositionY = 3;
+            _session.Character.PositionX = -13;
+            _session.Character.PositionY = -13;
```

Comment:
please create your own test don't change mine ;) 3/3 is still a wall and shouldnt work

Reply:
Yeah but Putitem randomly selects some cells around the character

Reply:
Yes this is why we use isblockedzone ;) and not iswalkable